schema/aggregations/filter_artworks_aggregation.js

```diff
@@ -24,17 +24,11 @@ export const ArtworksAggregation = new GraphQLEnumType({
     MEDIUM: {
       value: 'medium',
     },
-    FOR_SALE: {
-      value: 'for_sale',
-    },
     GALLERY: {
-      value: 'partner_id',
+      value: 'gallery',
     },
     INSTITUTION: {
-      value: 'partner_id',
-    },
-    RELATED_GENE: {
```

Comment:
Ha, I think I messed up by looking in the wrong place here (force instead of gravity)